mesheryctl/helpers/component_info.json

```diff
@@ -1,5 +1,5 @@
 {
   "name": "mesheryctl",
   "type": "client",
-  "next_error_code": 1060
+  "next_error_code": 2000
```

Comment:
Is there a reason for this large jump?

Reply:
Yes this error ERRO[0000] component_info.next_error_code '1060' is lower than or equal to highest used code '1079'

Reply:
there are more than 60 error code so it was needed

Reply:
Gotcha.